CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# 1.0.0 (February 24, 2022)
```

Comment:
When we actually do the release will we have more PRs for things like updating iOS version? Just wondering if we should use a single PR for all release things including changelog, version updates, etc. Or if we will have multiple PRs.